src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -363,75 +424,57 @@ private void onPageLoaded(boolean menuReplaced) {
    */
   private void scrollToHash() {
     String hash = Window.Location.getHash();
-    GQuery anchor = hash.length() > 1 ? $(hash + ", [name='" + hash.substring(1) + "']") : $();
-    if (anchor.isEmpty()) {
+    Element anchor = hash.length() > 1 ? document.querySelector(
+        hash + ", [name='" + hash.substring(1) + "']") : null;
+    if (anchor == null) {
       Window.scrollTo(0, 0);
     } else {
       anchor.scrollIntoView();
     }
   }
 
   private void maybeStyleHomepage() {
-    if ($("#content").hasClass("home")) {
+    if (document.querySelector("#content").classList.contains("home")) {
```

Comment:
```suggestion
    if (querySelector("#content").classList.contains("home")) {
```
right?

That said, could also be `querySelector("#content.home") != null" I think? Maybe harder to read...